bluepymm/run_combos/calculate_scores.py

```diff
@@ -218,9 +217,13 @@ def create_arg_list(scores_db_filename, emodel_dirs, final_dict,
         apical_points_isec = {}
         setup = tools.load_module('setup', emodel_dirs[one_row['emodel']])
         if hasattr(setup, 'multieval'):
-            apical_points_isec = tools.load_json(
-                os.path.join(one_row['morph_dir'], "apical_points_isec.json")
-            )
+            try:
+                apical_points_isec = tools.load_json(
+                    os.path.join(one_row['morph_dir'], "apical_points_isec.json")
```

Comment:
Don't you need the apical for MM?

Reply:
Well, the idea is that for the current release we don't use the bAP feature for the selection, so we don't need the values.
But I consider this fix fairly dangerous. I'd propose we add an option to the config file to actively disable the apical points